<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolDefinition</class>
 <widget class="QDialog" name="ToolDefinition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Milling Cutter And Holder Definition</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTabWidget" name="tabWidget2">
           <property name="minimumSize">
            <size>
             <width>325</width>
             <height>590</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>325</width>
             <height>590</height>
            </size>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="BallNose">
            <attribute name="title">
             <string>Ball Nose</string>
            </attribute>
            <layout class="QGridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Virmac.qrc">:/images/ballend_cutter.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Spherical">
            <attribute name="title">
             <string>Spherical</string>
            </attribute>
            <layout class="QGridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Virmac.qrc">:/images/spherical_cutter.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Toroidal">
            <attribute name="title">
             <string>Toroidal</string>
            </attribute>
            <layout class="QGridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Virmac.qrc">:/images/torical_cutter.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Square">
            <attribute name="title">
             <string>Square </string>
            </attribute>
            <layout class="QGridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Virmac.qrc">:/images/cylindrical_cutter.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="textLabel9">
               <property name="text">
                <string>Tool Number</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="toolNumber">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="textLabel1">
               <property name="text">
                <string>Holder Dia (HD).....</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="holderDia">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>45.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="textLabel2">
               <property name="text">
                <string>Holder Len (HL).....</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="holderLen">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>30.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="textLabel3">
               <property name="text">
                <string>Shank Dia (SD)......</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="shankDia">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>32.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="textLabel4">
               <property name="text">
                <string>Shank Len (SL)......</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="shankLen">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>35.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="textLabel5">
               <property name="text">
                <string>Taper Dia A (TDA).</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="taperDiaA">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>25.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="textLabel2_2">
               <property name="text">
                <string>Taper Dia B (TDB).</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="taperDiaB">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>32.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="textLabel6">
               <property name="text">
                <string>Taper Len (TL)......</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="taperLen">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>25.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="textLabel1_2">
               <property name="text">
                <string>BodyDia (BD)........</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="bodyDia">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>25.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="textLabel7">
               <property name="text">
                <string>Body Len (BL).......</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="bodyLen">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>30.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="textLabel8">
               <property name="text">
                <string>Ball Rad (R)...........</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="ballRad">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.001000000000000</double>
               </property>
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
               <property name="value">
                <double>12.500000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>250</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>345</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>&amp;Ok</string>
         </property>
         <property name="shortcut">
          <string>Alt+O</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
         <property name="shortcut">
          <string>Alt+C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="helpButton">
         <property name="text">
          <string>&amp;Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <resources>
  <include location="Virmac.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ToolDefinition</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ToolDefinition</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
